<launch>
  <!-- Set Parameter Sever -->
  <rosparam  file="$(find controller)/cfg/param_server.yaml" command="load"/>

  <!-- camera driver -->
  <node name="davis_ros_driver" pkg="davis_ros_driver" type="davis_ros_driver">
    <rosparam  file="$(find controller)/cfg/monstruck_camera_config.yaml" command="load"/>
  </node>

  <node name="dvs_renderer" pkg="dvs_renderer" type="dvs_renderer">
    <!-- <param name="display_method" value="grayscale"/> -->
    <param name="display_method" value="red-blue"/>
    <remap from="events" to="/dvs/events" />
    <remap from="image" to="/dvs/image_raw" />
    <remap from="camera_info" to="/dvs/camera_info" />
  </node>

</launch>
